migrations/000004_create_book_categories_table.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS book_categories (
+    book_id INT NOT NULL REFERENCES books(id) ON DELETE CASCADE,
```

Comment:
bigint

Reply:
int limitation is 2147483647, are you sure we would have more items?